provider/input_event.py

```diff
@@ -0,0 +1,575 @@
+import os
```

Comment:
Suggest the file name to be `input_event_proxy.py`

Reply:
updated, thanks~